tap_gitlab/schemas/vulnerabilities.json

```diff
@@ -0,0 +1,491 @@
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "dismissed_by_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "due_date": {
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "finding": {
+      "confidence": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "created_at": {
+        "anyOf": [
+          {
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "type": "null"
+          }
+        ]
+      },
+      "id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "location_fingerprint": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "metadata_version": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "name": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "primary_identifier_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "project_fingerprint": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "project_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "raw_metadata": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "report_type": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "scanner_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "severity": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "updated_at": {
+        "anyOf": [
+          {
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "type": "null"
+          }
+        ]
+      },
+      "uuid": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "vulnerability_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      }
+    },
+    "id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "last_edited_at": {
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "last_edited_by_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "project": {
+      "type": "object",
+      "properties": {
+        "created_at": {
+          "anyOf": [
+            {
+              "type": "string",
+              "format": "date-time"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "description": "",
```

Comment:
@benomahony is this a field of `project`?  If so this should be an object with type, etc. and not an empty string

Reply:
Fixed